index.d.ts

```diff
@@ -1,24 +1,26 @@
-import * as React from 'react'
-import { default as tippyCore, Instance, Props, Plugin } from 'tippy.js'
+import * as React from 'react';
+import {default as tippyCore, Instance, Props, Plugin} from 'tippy.js';
 
-type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
-export interface TippyProps extends Omit<Props, 'content'> {
-  content: React.ReactChild | React.ReactChild[]
-  children: React.ReactElement<any>
-  visible?: boolean
-  enabled?: boolean
-  className?: string
-  plugins?: Plugin[]
+export interface TippyProps extends Omit<Partial<Props>, 'content'> {
+  content: React.ReactChild | React.ReactChild[];
+  children: React.ReactElement<any>;
+  visible?: boolean;
+  enabled?: boolean;
+  className?: string;
+  plugins?: Plugin[];
 }
 
-declare const Tippy: React.ForwardRefExoticComponent<TippyProps>
-export default Tippy
+declare const Tippy: React.ForwardRefExoticComponent<TippyProps>;
+export default Tippy;
 
-export const tippy: typeof tippyCore
+export const tippy: typeof tippyCore;
 
 export interface TippySingletonProps extends Props {
```

Comment:
Add Partial here as it is only requried for the base interface. `children` is still required.
```suggestion
export interface TippySingletonProps extends Partial<Props> {
```